pkgs/games/openarena/default.nix

```diff
@@ -10,6 +10,13 @@ stdenv.mkDerivation {
     sha256 = "0jmc1cmdz1rcvqc9ilzib1kilpwap6v0d331l6q53wsibdzsz3ss";
   };
 
+  # https://openarena.fandom.com/wiki/OACMP/Volume_1
+  communityMaps = fetchurl {
+    name = "oacmp-volume1-v3.zip";
+    urls = ["https://archive.org/download/openarena/oacmp-volume1-v3.zip"];
```

Comment:
The download links from https://openarena.fandom.com/wiki/OACMP/Volume_1 are not stable, so I have uploaded this zip to archive.org.

Please could somebody download the original zip from https://openarena.fandom.com/wiki/OACMP/Volume_1 and confirm that the sha256 matches the sha256 in the PR.